docs/package.json

```diff
@@ -20,5 +20,8 @@
     "v-calendar": "^3.0.3",
     "yup": "^1.2.0",
     "zod": "^3.21.4"
+  },
+  "dependencies": {
+    "joi": "^17.9.2"
```

Comment:
Why do you add joi as dependency instead of devDependency ?

Reply:
That's a mistake, good catch!